client/modules/accounts/templates/tax-settings/taxSettings.html

```diff
@@ -0,0 +1,21 @@
+<template name="taxSettingsPanel">
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1 panel panel-default panel-tax-settings">
+      <h4 data-i18n="accounts.taxSettings">Tax Exempt Settings</h4>
+      <div>
+        {{#autoForm collection=Collections.Accounts schema=accountsSchema doc=account id=makeUniqueId type="update" }}
+          {{>afQuickField name='taxSettings.exemptionNo' class='form-control'}}
```

Comment:
add space after `{{` (I tried, it wouldn't let me update the file for some reason...